draft-ietf-git-using-github.md

```diff
@@ -428,6 +447,242 @@ A working group that uses GitHub MAY provide either facility at the request of
+
+Chairs and editors are given discretion in determining what issues are
+substantive.  As documents mature, it is generally prudent to err more toward
+consulting the mailing list where there is doubt.  As with other Working Group
+decisions, chairs are the arbiters in case of dispute.
+
+A recurrent problem with this mode of interaction is the tendency for
+discussions to spontaneously develop in the issue tracker.  This requires a
+degree of discipline from chairs and editors to ensure that any substantive
+matters are taken to the mailing list.
+
+As the Working Group mailing list remains the primary venue for discussion, this
+mode remains most compatible with existing work modes.  Participants in a
+Working Group that operates under this model can reasonably be expected to
+receive all relevant communication about the work of the group from the mailing
+list.
+
+Though the mailing list is used for making decisions, the issue tracker can
+still be a useful record of the state of issues.  It is often useful if chairs
+or editors record details of decisions in issue comments when closing issues as
+resolved.
+
+
+## Heavy GitHub Usage
+
+The "heavy" interaction mode differs from the "light" mode in that discussion
+relating to substantive technical matters is allowed to occur on GitHub issues.
+Though decisions are always subject to confirmation on the mailing list,
+participants are permitted to conduct substantive discussions on the issue
+tracker.  In some cases, this can include making some decisions without
+involving the Working Group mailing list.
+
+A Working Group mailing list remains a critical venue for decision making, even
+where issue discussion occurs elsewhere.  Working Group mailing lists generally
+include a wider audience than those who follow issue discussion, so difficult
+issues always benefit from list discussion.
+
+Decisions about Working Group consensus MUST always be confirmed using the
+Working Group mailing list.  However, depending on the maturity of documents,
+this might be a more lightweight interaction, such as sending an email
+confirmation for a set of resolutions made using GitHub.
+
+Using the mailing list to resolve difficult or controversial issues is strongly
+encouraged.  In those cases, the issue tracker might be used to more fully
+develop an understanding of problems before initiating a discussion on the
+mailing list, along lines similar to the design team process (see Section 6.5 of
+{{RFC2418}}).
+
+As a more involved process, adopting this mode can require changes in policies
+as documents become more mature.  It is possible to use a different process for
+each active document in the same Working Group.  Working Group chairs MUST
+ensure that the process that applies to different documents is clear, which
+includes both maintaining clear documentation about policies that relate to each
+document and providing suitable advance notice of changes to policies on the
+Working Group mailing list.
```

Comment:
the above paragraph lost me... it's just a bit circuituous. I'm not sure it's needed, but if so, I'd suggest making it very simple ("It is possible to use a different process for each active document in the same Working Group.").